pkg/kubelet/kuberuntime/kuberuntime_container_windows_test.go

```diff
@@ -165,7 +165,7 @@ func TestCalculateWindowsResources(t *testing.T) {
 			cpuLim: resource.MustParse("2"),
 			memLim: resource.MustParse("128Mi"),
 			expected: &runtimeapi.WindowsContainerResources{
-				CpuMaximum:         2500,
+				CpuMaximum:         10000 / int64(winstats.ProcessorCount()),
```

Comment:
Cherry-picked this change. Still seem to hit it:

```
=== RUN   TestCalculateWindowsResources
    kuberuntime_container_windows_test.go:193:
                Error Trace:    C:/Users/Administrator/workdir/kubernetes/pkg/kubelet/kuberuntime/kuberuntime_container_windows_test.go:193
                Error:          Not equal:
                                expected: &v1.WindowsContainerResources{CpuShares:0, CpuCount:0, CpuMaximum:5000, MemoryLimitInBytes:134217728, RootfsSizeInBytes:0, XXX_NoUnkeyedLiteral:struct {}{}, XXX_sizecache:0}
                                actual  : &v1.WindowsContainerResources{CpuShares:0, CpuCount:0, CpuMaximum:10000, MemoryLimitInBytes:134217728, RootfsSizeInBytes:0, XXX_NoUnkeyedLiteral:struct {}{}, XXX_sizecache:0}

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -3,3 +3,3 @@
                                  CpuCount: (int64) 0,
                                - CpuMaximum: (int64) 5000,
                                + CpuMaximum: (int64) 10000,
                                  MemoryLimitInBytes: (int64) 134217728,
                Test:           TestCalculateWindowsResources
--- FAIL: TestCalculateWindowsResources (0.00s)
```

Reply:
/remove-lgtm
Until this is fixed.